### ISSUE TITLE
BUG: Closures not working any more

### ISSUE THREAD

**Issue by JonathanDoelan** [optimistdigital/nova-settings#37]
Hello I’ve updated your package today and since then closures don’t work anymore:

```
// Using a callable
\OptimistDigital\NovaSettings\NovaSettings::addSettingsFields(function() {
  return [
    Text::make('Some setting', 'some_setting'),
    Number::make('A number', 'a_number'),
  ];
});
```

Inside Nova I’m getting „Closure object cannot have properties“

If I’m passing a class method, I’m getting “Undefined property: App\Nova\Settings\Settings::$panel” (where App\Nova\Settings\Settings is the class where all my fields are controlled)

**Comment by Tarpsvo** [comment]
Sec, looking into it.

**Comment by Tarpsvo** [comment]
Fixed in version 2.6.2 - there was indeed an issue with default path values. Sorry about that!
